### PR TITLE
Added retry button to setupv2

### LINKED ISSUE #1393: Add a manual Retry button to new repo creation flow
### Enhancement :sparkles:
Add a manual retry button to the `setupv2.html.erb` so that if enough goes wrong, a student has the option to perform a manual retry. 

This is useful when importing starter code fails but repo creation passes (since importing starter code is observed from `PorterStatusJob` but not retried).

In response to the post PR tasks in #1385

### PR DESCRIPTION
**Pull request by BenEmdon**
# What
This PR adds a retry button to the new import resiliency beta. This is what it looks like:

screen shot 2018-07-16 at 2 55 21 pm

The retry button is only visible on failure:
jul-16-2018 14-54-48

Closes #1393

### CODE REVIEW

File: app/assets/javascripts/setupv2.js [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+    indicate_completion,
+    indicate_failure,
+    indicate_in_progress,
+    invitation_path,
+    progress_path,
+    ready,
+    setup_retry_button,
+    show_success,
+    start_job,
+    success_path;
```
Comment (BenEmdon) [inline app/assets/javascripts/setupv2.js]: These are sorted by 🔤

File: app/assets/javascripts/setupv2.js [hunk 2]
```diff
@@ -54,6 +71,22 @@
     $(spinner_indicator).show();
     $(failure_indicator).hide();
     $(completion_indicator).hide();
+
+    hide_retry_button();
+  };
+
+  display_retry_button = function() {
+    var retry_button = $("#retry-button");
+    if (retry_button.hasClass("d-none")) {
```
Comment (srinjoym) [inline app/assets/javascripts/setupv2.js]: Might be nice to abstract the `if (foo.hasClass("d-none"` functionality into a function where you can pass into the object you're checking

Comment (srinjoym) [inline app/assets/javascripts/setupv2.js]: Might be nice to abstract the `if (foo.hasClass("d-none")` functionality into a function where you can pass in the object you're checking

**Comment by BenEmdon** [comment]
> To me, this feels a little bit more like a gray neutral button.

Sure, `gray btn` it is 👍

**Comment by BenEmdon** [comment]
### With gray button
screen shot 2018-07-17 at 10 39 30 am

**Pull request merged**